app/controllers/CheckYourAnswersController.scala

```diff
@@ -84,9 +86,16 @@ class CheckYourAnswersController @Inject()(identify: IdentifierActionProvider,
               )
               subscriptionDetails.subscriptionResponse match {
                 case subscribedResponse: SubscribedResponse =>
-                  enrolmentService.enrol(EnrolmentDetails(enrolmentKnownFacts, subscribedResponse.dprsId)).flatMap { _ =>
-                    sessionRepository.set(answersWithSubscription).map { _ =>
-                      Redirect(CheckYourAnswersPage.nextPage(NormalMode, answersWithSubscription))
+                  SendEmailRequest.build(request.userAnswers, subscribedResponse.dprsId).fold(
+                    errors => Future.failed(Exception(s"Unable to send email, path(s) missing: ${errors.toChain.toList.map(_.path).mkString(", ")}")),
```

Comment:
If we can't create the email, we should just not send the email and still try to enrol. Rather than erroring

Reply:
Fixed